src/api/support.ts

```diff
@@ -24,10 +37,13 @@ export async function getSupportCodeLibrary({
     requirePaths,
     importPaths,
   })
-  requireModules.map((module) => require(module))
-  requirePaths.map((path) => require(path))
+
+  requireModules.map((module) => tryRequire(module))
```

Comment:
Can we make the equivalent change for the parallel worker https://github.com/cucumber/cucumber-js/blob/main/src/runtime/parallel/worker.ts#L77-L78 as well?

(At some point it'd be good to extract some code common across these places, but not essential for this PR.)

Reply:
Done.  Moved the require logic to its own file in the same directory as the importer and named it try_require.  I see an opportunity for further refactoring, but I agree that should probably be another issue ticket.